platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -141,8 +149,14 @@ public ImportMigrationManagerImpl(
       this.currentProductBranding = currentProductBranding;
       this.currentProductVersion = currentProductVersion;
       // process migratables' metadata
-      JsonUtils.getMapFrom(metadata, MigrationContextImpl.METADATA_MIGRATABLES)
+      JsonUtils.getMapFrom(metadata, MigrationContextImpl.METADATA_MIGRATABLES, false)
           .forEach((id, o) -> getContextFor(id).processMetadata(JsonUtils.convertToMap(o)));
+      if (MigrationContextImpl.NEED_SYSTEM_PROPERTIES_VERSIONS.contains(productVersion)) {
```

Comment:
❗️ This will not work with upstream projects that have different versions.

Reply:
I made it conditional because @paouelle suggested that I not set it up if not necessary. I see the problem you mention though. Should I change it back to what I had before? https://github.com/codice/ddf/pull/5540#discussion_r345790334